server/server_test.go

```diff
@@ -437,6 +438,63 @@ func (cli *testServerClient) runTestLoadDataWithSelectIntoOutfile(c *C, server *
 		}
 	})
 }
+func (cli *testServerClient) runTestLoadDataForSlowLog(c *C, server *Server) {
+	path := "/tmp/load_data_test.csv"
```

Comment:
This path makes the Windows user upset, maybe this one is better https://pkg.go.dev/os#TempDir